app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id'
+  belongs_to :post
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- **[OPTIONAL]** It would be a good to idea to add an `after_save` active record callback for this method.